data/referrer_mapping.csv

```diff
@@ -55,7 +55,6 @@ social,Hocam.com,hocam.com
 social,Hyves,hyves.nl
 social,Taringa!,taringa.net
 social,Classmates,classmates.com
-social,Pinterest,pinterest.com

```

Comment:
I'm not clear on why this line got removed

Reply:
No worries! Line 58 is a duplicate of 1755 (when the pinterest category got expanded). If they are not unique, it will cause a duplicate join which prevents uniqueness in the warehouse. So I removed it, because this was added later in 1755.